src/Dscp/Slotting.hs

```diff
@@ -0,0 +1,33 @@
+-- | Minimalistic slotting. Currently we only need to get uniform
+-- slots numbers so that several nodes can issue blocks one after
+-- another.
+
+module Dscp.Slotting
+       ( getCurrentSlot
+       , waitUntilNextSlot
+       ) where
+
+import Control.Concurrent (threadDelay)
+import Data.Time.Clock.POSIX (getPOSIXTime)
+
+newtype SlotId = SlotId Integer deriving (Eq, Ord, Show, Generic)
+
+-- TODO pass it instead of fixing.
+-- In microseconds.
+slotLength :: Integer
+slotLength = 20000000 -- 20 sec
```

Comment:
Why not `DiffTime`, for instance?

Reply:
It's a stupid stub for now, we'll improve it when it'll come to real slotting. This PR forces me to implement a lot of different components which are not directly related to what I do, so they are sometimes naive and simple.